commit fff384555f92e4849e6703bca68b8ecb82c67b07
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Tue Feb 6 13:10:09 2018 +0000

    BCM270X_DT: More cosmetic DT changes
    
    Remove unnecessary duplicate labels (but keep spi0).
    
    Signed-off-by: Phil Elwell <phil@raspberrypi.org>

diff --git a/arch/arm/boot/dts/bcm270x.dtsi b/arch/arm/boot/dts/bcm270x.dtsi
index 8fba2e245a09..0e421a8f79bc 100644
--- a/arch/arm/boot/dts/bcm270x.dtsi
+++ b/arch/arm/boot/dts/bcm270x.dtsi
@@ -12,10 +12,6 @@
 			/* Add alias */
 		};
 
-		cprman: cprman@7e101000 {
-			/* Add alias */
-		};
-
 		random: rng@7e104000 {
 			/* Add alias */
 		};
@@ -106,7 +102,7 @@
 			reg = <0x7e600000 0x100>;
 			interrupts = <2 16>;
 			clocks = <&clocks BCM2835_CLOCK_SMI>;
-			assigned-clocks = <&cprman BCM2835_CLOCK_SMI>;
+			assigned-clocks = <&clocks BCM2835_CLOCK_SMI>;
 			assigned-clock-rates = <125000000>;
 			dmas = <&dma 4>;
 			dma-names = "rx-tx";
@@ -136,11 +132,6 @@
 			status = "disabled";
 		};
 
-		gpu: gpu {
-			/* Add alias */
-			status = "disabled";
-		};
-
 		i2c_dsi: i2cdsi {
 			/* We have to use i2c-gpio because the
 			 * firmware is also polling another device
@@ -192,3 +183,7 @@
 &dsi1 {
 	power-domains = <&power RPI_POWER_DOMAIN_DSI1>;
 };
+
+&vc4 {
+	status = "disabled";
+};
diff --git a/arch/arm/boot/dts/overlays/vc4-fkms-v3d-overlay.dts b/arch/arm/boot/dts/overlays/vc4-fkms-v3d-overlay.dts
index 36fbf6c8c2e6..14f7687172c3 100644
--- a/arch/arm/boot/dts/overlays/vc4-fkms-v3d-overlay.dts
+++ b/arch/arm/boot/dts/overlays/vc4-fkms-v3d-overlay.dts
@@ -66,7 +66,7 @@
 	};
 
 	fragment@8 {
-		target = <&gpu>;
+		target = <&vc4>;
 		__overlay__  {
 			status = "okay";
 		};
diff --git a/arch/arm/boot/dts/overlays/vc4-kms-v3d-overlay.dts b/arch/arm/boot/dts/overlays/vc4-kms-v3d-overlay.dts
index bf9395041fac..b6df001b03b8 100644
--- a/arch/arm/boot/dts/overlays/vc4-kms-v3d-overlay.dts
+++ b/arch/arm/boot/dts/overlays/vc4-kms-v3d-overlay.dts
@@ -53,20 +53,13 @@
 	};
 
 	fragment@6 {
-		target = <&cprman>;
-		__overlay__  {
-			status = "okay";
-		};
-	};
-
-	fragment@7 {
 		target = <&fb>;
 		__overlay__  {
 			status = "disabled";
 		};
 	};
 
-	fragment@8 {
+	fragment@7 {
 		target = <&pixelvalve0>;
 		__overlay__  {
 			interrupts = <2 13>; /* pwa0 */
@@ -74,7 +67,7 @@
 		};
 	};
 
-	fragment@9 {
+	fragment@8 {
 		target = <&pixelvalve1>;
 		__overlay__  {
 			interrupts = <2 14>; /* pwa1 */
@@ -82,7 +75,7 @@
 		};
 	};
 
-	fragment@10 {
+	fragment@9 {
 		target = <&pixelvalve2>;
 		__overlay__  {
 			interrupts = <2 10>; /* pixelvalve */
@@ -90,7 +83,7 @@
 		};
 	};
 
-	fragment@11 {
+	fragment@10 {
 		target = <&hvs>;
 		__overlay__  {
 			interrupts = <2 1>;
@@ -98,7 +91,7 @@
 		};
 	};
 
-	fragment@12 {
+	fragment@11 {
 		target = <&hdmi>;
 		__overlay__  {
 			interrupts = <2 8>, <2 9>;
@@ -106,7 +99,7 @@
 		};
 	};
 
-	fragment@13 {
+	fragment@12 {
 		target = <&v3d>;
 		__overlay__  {
 			interrupts = <1 10>;
@@ -114,21 +107,21 @@
 		};
 	};
 
-	fragment@14 {
-		target = <&gpu>;
+	fragment@13 {
+		target = <&vc4>;
 		__overlay__  {
 			status = "okay";
 		};
 	};
 
-	fragment@15 {
+	fragment@14 {
 		target-path = "/soc/dma";
 		__overlay__ {
 			brcm,dma-channel-mask = <0x7f35>;
 		};
 	};
 
-	fragment@16 {
+	fragment@15 {
 		target = <&dsi1>;
 		__overlay__  {
 			#address-cells = <1>;
@@ -143,21 +136,21 @@
 		};
 	};
 
-	fragment@17 {
+	fragment@16 {
 		target = <&i2c_dsi>;
 		__overlay__  {
 			status = "okay";
 		};
 	};
 
-	fragment@18 {
+	fragment@17 {
 		target = <&vec>;
 		__overlay__  {
 			status = "okay";
 		};
 	};
 
-	fragment@19 {
+	fragment@18 {
 		target = <&clocks>;
 		__overlay__  {
 			claim-clocks = <
